packages/boxel-ui/addon/src/components/select/trigger.gts

```diff
@@ -2,6 +2,7 @@ import Component from '@glimmer/component';
 import type { Select } from 'ember-power-select/components/power-select';
```

Comment:
Changes in this component was to remove the arrow that suggest dropdown when the component was disabled